lib/index.js

```diff
@@ -192,6 +192,19 @@ function sensitiveDataAudit () {
         });
 }
 
+function runNSPCheck () {
+    var stopSpinner = createSpinner('Performing nsp check');
+    // need to target the local nsp
+    return cmd('./node_modules/.bin/nsp check')
+         .catch(function () {
+            stopSpinner(false);
+            throw "NSP check failed";
```

Comment:
Can we add some more detailed error messages from this thing? Like, what package triggered the fail and so on.